demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -14,4 +14,8 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
         { type: "date", name: "availableSince", width: 140 },
         { type: "dateTime", name: "createdAt", width: 170 },
     ],
+    actions: {
```

Comment:
Have you considered having `type: "actions"` in the `columns`? This would allow overriding the default actions and placing the actions anywhere in the grid. Do we even want that?

Reply:
I also think this would be natural, as it's a column after all

Reply:
@jamesricky what about this suggestion?

Reply:
I agree, this api is better. 
Changed it in c972a96b68cbb396facc5e795d16c8ba3bd2ca69